src/EFCore.PG/NpgsqlRangeExtensions.cs

```diff
@@ -0,0 +1,253 @@
+﻿#region License
+
+// The PostgreSQL License
+//
+// Copyright (C) 2016 The Npgsql Development Team
+//
+// Permission to use, copy, modify, and distribute this software and its
+// documentation for any purpose, without fee, and without a written
+// agreement is hereby granted, provided that the above copyright notice
+// and this paragraph and the following two paragraphs appear in all copies.
+//
+// IN NO EVENT SHALL THE NPGSQL DEVELOPMENT TEAM BE LIABLE TO ANY PARTY
+// FOR DIRECT, INDIRECT, SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES,
+// INCLUDING LOST PROFITS, ARISING OUT OF THE USE OF THIS SOFTWARE AND ITS
+// DOCUMENTATION, EVEN IF THE NPGSQL DEVELOPMENT TEAM HAS BEEN ADVISED OF
+// THE POSSIBILITY OF SUCH DAMAGE.
+//
+// THE NPGSQL DEVELOPMENT TEAM SPECIFICALLY DISCLAIMS ANY WARRANTIES,
+// INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+// AND FITNESS FOR A PARTICULAR PURPOSE. THE SOFTWARE PROVIDED HEREUNDER IS
+// ON AN "AS IS" BASIS, AND THE NPGSQL DEVELOPMENT TEAM HAS NO OBLIGATIONS
+// TO PROVIDE MAINTENANCE, SUPPORT, UPDATES, ENHANCEMENTS, OR MODIFICATIONS.
+
+#endregion
+
+using System;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Query.ExpressionTranslators;
+using NpgsqlTypes;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL
+{
+    /// <summary>
+    /// Provides extension methods for <see cref="NpgsqlRange{T}"/> supporting PostgreSQL translation.
+    /// </summary>
+    public static class NpgsqlRangeExtensions
+    {
+        /// <summary>
+        /// Determines whether a range contains a specified value.
+        /// </summary>
+        /// <param name="range">
```

Comment:
nit: here and below, why not put the `<param>` and `<typeparam>` in a single line:
```c#
/// <param name="range">The range in which to locate the value</param>
```
This seems to be standard in the .NET / EF Core code bases and reduces lots of useless lines